src/components/schema-ui/schema-menu-attribute.js

```diff
@@ -1,18 +1,70 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
+import {FieldFocusContext} from "../../contexts/field-focus";
 
 const SchemaMenuAttributeName = ({ name }) => (
   <div className="menu__attribute">
     <span className="link__title link__title--readable">{name}</span>
   </div>
 );
 
+const AttributeFocusToggle = ({path}) => {
+  const { focusPath, setFocusPath, availableFocusPaths } = useContext(FieldFocusContext);
+  const [ lastFocusPath, setLastFocusPath ] = useState(focusPath);
+  const [ pinned, setPinned ] = useState(false);
+  const [ unpinned, setUnpinned ] = useState(false);
+
+  const onMouseEnter = () => {
```

Comment:
I'm glad you added this- I thought it was a cool idea when we discussed it. I found it to be a little unresponsive though, not sure exactly what's going on but the mouseEnter and MouseLeave events were unreliable.

Reply:
Hm, was totally fine for me. I'll see what I can do.

Reply:
Ohh, I bet it's because I only have 2 nodes and you have dozens from devel gen.